### PR TITLE
feat(ng) ADD Title service

### PR DESCRIPTION
**Pull request by shajz**
Adds utility service that sets the page title according to routing rules. It respects Lucca's rules for page titles ("Current page title - Current app - Lucca" i.e. "Purchases - Cleemy Procurement - Lucca")

Inspired from https://github.com/LuccaSA/Cleemy.Front/blob/master/projects/cleemy/page-title/src/page-title.service.ts#L11
Thanks to @belabb-z @jbosi @mathieu-bellange @JeSuisCharlie1 @LucasPerrault @albangranger 

fixes https://github.com/LuccaSA/Guildes/issues/48

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

File: packages/ng/title/src/lib/title.service.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { ActivatedRouteSnapshot, ActivationEnd, Router } from '@angular/router';
+import { BehaviorSubject } from 'rxjs';
+import { distinctUntilChanged, filter, map, tap } from 'rxjs/operators';
+import { ILuTitleTranslateService, LU_TITLE_TRANSLATE_SERVICE } from './title-translate.service';
+
+export type PageTitle = { title: string; params: { [param: string]: string } };
+export const TitleSeparator = ' – ';
+
+@Injectable()
+export class TitleService {
```
Comment (jbosi) [inline packages/ng/title/src/lib/title.service.ts:12]: On parle de `LuTitleModule` mais le `TitleService` n'a pas le préfixe `Lu`, à modifier ?

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1650

**Review by jbosi — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Review by t8g — APPROVED** [review]

**Pull request merged**